### PR TITLE
LCI first 5 cards

### PR DESCRIPTION
**Pull request by Simisays**

### CODE REVIEW

File: forge-gui/res/cardsfolder/upcoming/huatli_poet_of_unity_roar_of_the_fifth_people.txt [hunk 1]
```diff
@@ -0,0 +1,27 @@
+Name:Huatli, Poet of Unity
+ManaCost:2 G
+Types:Legendary Creature Human Warrior Bard
+PT:2/3
+T:Mode$ ChangesZone | Origin$ Any | Destination$ Battlefield | ValidCard$ Card.Self | Execute$ TrigChange | TriggerDescription$ When CARDNAME enters the battlefield, search your library for a basic land card, reveal it, put it into your hand, then shuffle.
+SVar:TrigChange:DB$ ChangeZone | Origin$ Library | Destination$ Hand | ChangeType$ Land.Basic | ChangeNum$ 1
+A:AB$ ChangeZone | Cost$ 3 RW RW | Origin$ Battlefield | Destination$ Exile | RememberChanged$ True | SubAbility$ DBReturn | SorcerySpeed$ True | StackDescription$ SpellDescription | SpellDescription$ Exile CARDNAME, then return it to the battlefield transformed under its owner's control.
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/huatli_poet_of_unity_roar_of_the_fifth_people.txt:7]: NICKNAME

Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/huatli_poet_of_unity_roar_of_the_fifth_people.txt:7]: why SorcerySpeed

Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/huatli_poet_of_unity_roar_of_the_fifth_people.txt:7]: description incomplete

File: forge-gui/res/cardsfolder/upcoming/huatli_poet_of_unity_roar_of_the_fifth_people.txt [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+SVar:DBReturn:DB$ ChangeZone | Defined$ Remembered | Origin$ Exile | Destination$ Battlefield | Transformed$ True | SubAbility$ DBCleanup
+SVar:DBCleanup:DB$ Cleanup | ClearRemembered$ True
+DeckHas:Ability$Token Type$Dinosaur
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/huatli_poet_of_unity_roar_of_the_fifth_people.txt:10]: typo

File: forge-gui/res/cardsfolder/upcoming/the_skullspore_nexus.txt [hunk 3]
```diff
@@ -0,0 +1,12 @@
+Name:The Skullspore Nexus
+ManaCost:6 G G
+Types:Legendary Artifact
+S:Mode$ ReduceCost | ValidCard$ Card.Self | Type$ Spell | Amount$ X | EffectZone$ All | Description$ CARDNAME costs {X} less to cast, where X is the greatest power among creatures you control.
+SVar:X:Count$Valid Creature.YouCtrl$GreatestPower
+T:Mode$ ChangesZoneAll | ValidCards$ Creature.nonToken+YouCtrl | Origin$ Battlefield | Destination$ Graveyard | TriggerZones$ Battlefield | Execute$ TrigToken | TriggerDescription$ Whenever one or more nontoken creatures you control die, create a green Fungus Dinosaur creature token with base power and toughness each equal to the total power of those creatures.
+SVar:TrigToken:DB$ Token | TokenScript$ g_x_x_fungus_dinosaur | TokenPower$ Z | TokenToughness$ Z
+SVar:Z:TriggerObjects$CardPower
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/the_skullspore_nexus.txt:8]: TriggerObjectsCards...

File: forge-gui/res/cardsfolder/upcoming/ojer_axonil_deepest_might.txt.txt [hunk 4]
```diff
@@ -0,0 +1,23 @@
+Name:Ojer Axonil, Deepest Might
+ManaCost:2 R R
+Types:Legendary Creature God
+PT:4/4
+K:Trample
+R:Event$ DamageDone | ActiveZones$ Battlefield | ValidSource$ Card.RedSource+YouCtrl | IsCombat$ False | ValidTarget$ Player | ReplaceWith$ DmgPlus | Description$ If a red source you control would deal an amount of noncombat damage less than NICKNAME's power to an opponent, that source deals damage equal to CARDNAME's power instead.
+SVar:DmgPlus:DB$ ReplaceEffect | ConditionCheckSVar$ X | ConditionCompare$ LTY | VarName$ DamageAmount | VarValue$ Y
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/ojer_axonil_deepest_might.txt.txt:7]: use DamageAmount$

File: forge-gui/res/cardsfolder/upcoming/ojer_axonil_deepest_might.txt.txt [hunk 5]
```diff
@@ -0,0 +1,23 @@
+Name:Ojer Axonil, Deepest Might
+ManaCost:2 R R
+Types:Legendary Creature God
+PT:4/4
+K:Trample
+R:Event$ DamageDone | ActiveZones$ Battlefield | ValidSource$ Card.RedSource+YouCtrl | IsCombat$ False | ValidTarget$ Player | ReplaceWith$ DmgPlus | Description$ If a red source you control would deal an amount of noncombat damage less than NICKNAME's power to an opponent, that source deals damage equal to CARDNAME's power instead.
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/ojer_axonil_deepest_might.txt.txt:6]: wrong target

File: forge-gui/res/cardsfolder/upcoming/ojer_axonil_deepest_might.txt.txt [hunk 6]
```diff
@@ -0,0 +1,22 @@
+Name:Ojer Axonil, Deepest Might
+ManaCost:2 R R
+Types:Legendary Creature God
+PT:4/4
+K:Trample
+R:Event$ DamageDone | ActiveZones$ Battlefield  | DamageAmount$ LTX | ValidSource$ Card.RedSource+YouCtrl | IsCombat$ False | ValidTarget$ Opponent | ReplaceWith$ DmgPlus | Description$ If a red source you control would deal an amount of noncombat damage less than NICKNAME's power to an opponent, that source deals damage equal to CARDNAME's power instead.
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/ojer_axonil_deepest_might.txt.txt:6]: NICKNAME

Comment (Hanmac) [reply forge-gui/res/cardsfolder/upcoming/ojer_axonil_deepest_might.txt.txt]: Also filename is".txt.txt"

**Review by tool4ever — APPROVED** [review]

**Pull request merged**